libs/stats/odc/stats/cli.py

```diff
@@ -3,9 +3,12 @@
 from tqdm.auto import tqdm
 import sys
 import pickle
+import pandas as pd
+from typing import Dict, Tuple, List
 from datetime import datetime
 from collections import namedtuple
 from odc.io.text import click_range2d
+from datacube.utils.dates import normalise_dt
```

Comment:
can you move this into function, `datacube` imports are slow and really slow down cli, same for pandas

Reply:
Moved binning of data to a separate file.